crf/src/main/assets/json/stair_step.json

```diff
@@ -134,16 +134,16 @@
     {
       "identifier"       : "hr",
       "type"             : "crf_heart_rate_camera_step",
-      "hideProgress"     : "true",
+      "isHrRecoveryStep" : "true",
       "duration"         : "60"
     },
     {
       "identifier"       : "completion",
       "type"             : "crf_completion",
-      "topText"          : "Your heart rate recovery is",
+      "topText"          : "Your estimate Recovery HR score is",
       "text"             : "Recovery HR Score",
-      "detailText"       : "Changing your Recovery HR can take time. You can change it over weeks or months by losing weight or exercising.",
-      "learnMoreText"    : "What is a Recovery HR Score?",
+      "detailText"       : "The score above estimates a vital sign called VO2max that is normally measured in a lab or clinic. If you took that lab-based test, it may be between:",
```

Comment:
"The score above estimates a vital sign called VO2max that is normally measured in a lab or clinic."